contracts/marketplace/Marketplace.sol

```diff
@@ -197,16 +202,29 @@ contract Marketplace is Ownable, Pausable, Destructible {
     );
 
     // Transfer asset owner
-    nonFungibleRegistry.safeTransferFrom(
+    nftRegistry.safeTransferFrom(
```

Comment:
This is not related to the PR, but what happens if a user publishes let say 10 assets and then disapproves the contract? All calls will fail here, which is fine, maybe is something to manage in the frontend.